python_modules/automation/parse_spark_configs.py

```diff
@@ -18,6 +19,138 @@
 TABLE_REGEX = r"### (.{,30}?)\n\n(<table.*?>.*?<\/table>)"
 
 
+class CONFIG_TYPE(Enum):
```

Comment:
we'll generally been doing camelcase (ConfigType) for the enum classes themselves

Reply:
cool, fixed